Lib/test/test_socket.py

```diff
@@ -2277,6 +2272,26 @@ def testFromFd(self):
     def _testFromFd(self):
         self.serv_conn.send(MSG)
 
+    @requireAttrs(socket.socket, 'send_fds')
+    def testSendAndRecvFds(self):
+        fds = []
+        for i in range(1):
```

Comment:
Why we need a loop of 1 iteration?

Reply:
I want to return a list of 1 file descriptors for created files.

Reply:
But you don't need a loop for that, right? Just substitute i by 0. I am missing something?

Reply:
But you don't need a loop for that, right? Just substitute i by 0. I am missing something?

Reply:
Am actually just reusing this method : https://github.com/python/cpython/pull/12889/files#diff-30065f853d7944a19b3e17b55db67733R2213